spec/support/factories.rb

```diff
@@ -6,6 +6,13 @@
     password "password"
   end
 
+  factory :admin, class: User do
```

Comment:
You can change the value of admin when you call FactoryGirl:
```
let(:admin) { FactoryGirl.create(:user, admin: true) }
```
would replace your current code below:
```
let(:admin) { FactoryGirl.create(:admin) }
```